### ISSUE TITLE
Title is not moving along with the scrolling of list

### ISSUE THREAD

**Issue by HimanshuS01** [systers/PCSA-Android#393]
Title is not moving along with  the scrolling of list in **SafetyTools--> Unwanted attention** & in **SupportServices-->PeaceCopse MythBusters**. 
![ezgif com-video-to-gif 7](https://cloud.githubusercontent.com/assets/18090596/23550872/59fa78de-0039-11e7-8f17-526f35222918.gif)
![ezgif com-video-to-gif 6](https://cloud.githubusercontent.com/assets/18090596/23550875/5b5aeb3c-0039-11e7-9000-f8d82f1f9dd1.gif)

**Fix:**
Title must also be scrolled up along with the list ,similar like in **SupportServices-->What To do after an assault-->Reporting Steps** to maintain uniformity in app.
Scrolling along with the  title should be like this:-
![ezgif com-video-to-gif 3](https://cloud.githubusercontent.com/assets/18090596/23500614/b5d6f666-ff56-11e6-9981-967f5bbdd8b0.gif)

This issue is one of the part of #387 as I were asked to create the separate issue.

@sandarumk can I work on this issue and send the PR ?
Thank you

**Comment by sandarumk** [comment]
In Peace corps Mythbusters, it is incorrect to have the title as Potential Response Strategies.  It should be Assumptions and Facts. You can change this as a first step and send a PR. It is certain that the app should follow a uniform set up. Can I know why you think that moving the title is the correct behaviour and not vice versa?

**Comment by HimanshuS01** [comment]
yeah @sandarumk sure. I will change the title to Assumptions and Facts

Also for the above issue,title is generally scrolling at the rest of the places in an application along with the layout.For Example in **SupportServices-->What To do After an assault-->Reporting Steps** as well as in **Sexual Assault Awareness-->Helping Others**.

So What I thought was, Title in **SafetyTools--> Unwanted attention** and in **SupportServices-->PeaceCopse MythBusters** should also move along with the layout so as to maintain similarity and uniformity.

@sandarumk what are your view's  ? Can I proceed with this ?
_Thank you_
